### PR TITLE
First submit of JS2WEEK1 Homework

### PR DESCRIPTION
**Pull request by shanhaider23**

### CODE REVIEW

File: javascript/javascript2/week1/ex_01.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+console.log(findDanishLetters(danishString2));
+console.log(findDanishLetters(danishString));
+// Spirit animal name generator
+
+const differentAnimals = ["Squirrel", "Tiger", "Chimpanzee","Lion", "Panda" , "Walrus"];
+const inputName = document.querySelector("#userName");
+const entered = document.querySelector("#submit");
+const outputAnimal = document.getElementById("writeAnimal");
+entered.addEventListener("click", getSpiritAnimalName);
```
Comment (cameronjdavis) [inline javascript/javascript2/week1/ex_01.js:58]: This code works but can you also make it so when the user's mouse goes over the input text field then a new name is generated. See the `mouseover` event.

File: javascript/javascript2/week1/main.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    }
+}
+
+renderProducts(products); 
+
+
+
+
+
+
```
Comment (cameronjdavis) [inline javascript/javascript2/week1/main.js]: Clean up these extra lines. Clean code is easier to read.

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript2/week1/ex_01.js (hunk 1 above)
Comment (shanhaider23) [reply javascript/javascript2/week1/ex_01.js:58]: Thanks, I tried to do it I am not sure it is ok?

File: javascript/javascript2/week1/main.js (hunk 2 above)
Comment (shanhaider23) [reply javascript/javascript2/week1/main.js]: ok I will

**Pull request merged**